docs/GlobalAPI.md

```diff
@@ -0,0 +1,332 @@
+---
+id: api
+title: Globals
+layout: docs
+category: API Reference
+permalink: docs/api.html
+next: expect
+---
+
+In your test files, Jest puts each of these methods and objects into the global environment. You don't have to require or import anything to use them.
```

Comment:
Ah - disregard my earlier comment then. 

We'll need to [update the intro](https://github.com/facebook/jest/pull/2388#pullrequestreview-13866891) in `JestObjectAPI.md`.